scipy/fft/__init__.py

```diff
@@ -44,6 +44,15 @@
    dstn - N-D Discrete sine transform
    idstn - N-D Inverse discrete sine transform
 
+Fast Hankel Transforms
+======================
+
+.. autosummary::
+    :toctree: generated/
+
+    fht - Fast Hankel transform
+    ifht - Inverse of fht
```

Comment:
The CI failure looks related I suppose?

```
Checking tutorial files at doc/source/tutorial/*.rst:

=========
scipy.fft
=========

Non-deprecated objects in __all__: 40
Objects in refguide: 38

ERROR: objects in scipy.fft.__all__ but not in refguide::

    fht
    ifht

This issue can be fixed by adding these objects to
the function listing in __init__.py for this module
```

Reply:
Isn't this the function listing in __init__.py?